hw/xbox/smbus_xbox_smc.c

```diff
@@ -189,6 +216,14 @@ static void smbus_smc_realize(DeviceState *dev, Error **errp)
     if (object_property_get_bool(qdev_get_machine(), "short-animation", NULL)) {
         smc->scratch_reg = SMC_REG_SCRATCH_SHORT_ANIMATION;
     }
+
+    avpack = object_property_get_str(qdev_get_machine(), "avpack", NULL);
```

Comment:
`object_property_get_str` gives us ownership of the string returned, so we should `g_free` it if we no longer need it. Side note: we should also fix this in the other places we call `object_property_get_str` as I'm pretty sure we leak it in exactly the same way.

Reply:
Addressed. :ok_hand: